### ISSUE TITLE
AsusCertService needs to be disabled after other services.

### ISSUE THREAD

**Issue by MrAS04** [seerge/g-helper#814]
**Describe the bug**
AsusCertService is one of the services that are disabled by GHelper through the extras menu. In version 0.98.0.0 and my model, disabling AsusCertService errors out with `Set-Service : Cannot stop service 'AsusCertService (AsusCertService)' because it is dependent on other services.` as shown in the attached logs. Using the `Set-Service -Name "AsusCertService" -Status stopped -StartupType disabled` command in an Administrator PowerShell after GHelper disables all other services is successful.

**Clear scenario to Reproduce**
1. Go to GHelper.
2. Click on the "Extra" tab in the Laptop Keyboard section.
3. Re-enable all ASUS services.
4. Attempt to disable all ASUS services again.
5. One Asus Service is still running.

**Expected behavior**
All ASUS services should be stopped when the Stop button is clicked.

**App Logs**
[GHelper log.txt](https://github.com/seerge/g-helper/files/12089417/GHelper.log.txt)
Error is visible on line 1013.

**Desktop (please complete the following information):**
 - OS: Windows 11 22H2
 - Laptop model: ASUS Zephyrus M16 GU603HR

**Asus software**
- Armoury crate (or it's services installed): Removed.
- MyASUS installed: Yes.
- Other Asus services running in background: Always kept disabled using GHelper.

**Comment by seerge** [comment]
@MrAS04 i have set it to be last in the list, does it help?
[GHelper.zip](https://github.com/seerge/g-helper/files/12089427/GHelper.zip)

**Comment by MrAS04** [comment]
Hmm, it unfortunately still seems to error out with the same issue. Not exactly sure why.

By the way, clicking "Stop" again when there is only 1 service running results in it restarting all services and re-opening GHelper. This shouldn't be a issue if all services are correctly starting and stopping though.

Here's the logs.
[GHelper Logs V2.txt](https://github.com/seerge/g-helper/files/12089466/GHelper.Logs.V2.txt)

**Comment by seerge** [comment]
@MrAS04 well, i can't really reproduce it. I have moved optimization / armoury-button services to the beginning of kill-list, mb that will help ?
[GHelper.zip](https://github.com/seerge/g-helper/files/12093302/GHelper.zip)

Otherwise, try to figure out what exactly is blocking your cert service :)

P.S. Just in case cert-service is absolutely not critical for anything, it's in the least just to keep things tidy

**Comment by MrAS04** [comment]
Hey, so your new version works*!

*However, there was one error after I used the previous version to stop all services (with one service remaining) and then tried to use the new version to close them all. It tried to stop all services but was stuck for a very long time until the logs showed it timed out. Looking through the logs and using `Get-Service` to investigate, the `AsusLinkNear` service was stuck in the `StopPending` state. Googling tells me that to fix this issue, the general advice is just to restart which probably does fix the issue however [this blog post](https://mikefrobbins.com/2013/08/29/powershell-function-to-stop-a-windows-service-with-a-status-of-stopping/) has a relatively simple function that they defined that gets the service's PID and stops that using `Stop-Process` which then allows `Stop-Service` to work properly. Using that function, I was the able to get `AsusLinkNear` unstuck and then close it. Afterwards, I was able to stop and restart all processes via GHelper with no problem. [Here are the logs.](https://github.com/seerge/g-helper/files/12100338/GHelper.Logs.3.txt)

As this is unlikely to be a persistent bug (should only happen updating from the previous version to this newer one, I think?), I think I should close this issue as this newest version fixes the problem! Thanks very much for your quick response and fix!
